### PR TITLE
Added JSON syntax highlightning to the Obsidian theme

### PR DESCRIPTION
**Pull request by victor-istomin**
Hi, 

It looks like the Obsidian theme lacks JSON LexerType, so JSON syntax highlighting is gone when switching to it. 
I'd like to fix the inconvenience with this pull-request. 

image
Also, the JSON scheme is missing from most custom themes, except the 'DansLeRuSH-Dark.xml' and default 'stylers.xml', but I'm not in time to craft decent colors for them right now, so I'd like to fix Obsidian only. 

Thanks!

### CODE REVIEW

File: PowerEditor/installer/themes/Obsidian.xml [hunk 1]
```diff
@@ -758,6 +758,22 @@ Notepad++ Custom Style
...
+            <WordsStyle name="LINECOMMENT" styleID="6" fgColor="008000" bgColor="293134" fontName="" fontStyle="0" fontSize="" />
+            <WordsStyle name="BLOCKCOMMENT" styleID="7" fgColor="008000" bgColor="293134" fontName="" fontStyle="0" fontSize="" />
+            <WordsStyle name="OPERATOR" styleID="8" fgColor="E8E2B7" bgColor="293134" fontName="" fontStyle="0" fontSize="" />
+            <WordsStyle name="URI" styleID="9" fgColor="EC7600" bgColor="293134" fontName="" fontStyle="0" fontSize="" />
+            <WordsStyle name="COMPACTIRI" styleID="10" fgColor="0000A0" bgColor="293134" fontName="" fontStyle="1" fontSize="" />
+            <WordsStyle name="KEYWORD" styleID="11" fgColor="93C763" bgColor="293134" fontName="" fontStyle="1" fontSize="" keywordClass="instre1" />
+            <WordsStyle name="LDKEYWORD" styleID="12" fgColor="FF0000" bgColor="293134" fontName="" fontStyle="0" fontSize="" keywordClass="instre2" />
+            <WordsStyle name="ERROR" styleID="13" fgColor="FFFF80" bgColor="FF0000" fontName="" fontStyle="0" fontSize="" />
```
Comment (mere-human) [inline PowerEditor/installer/themes/Obsidian.xml:775]: Could you, please, change it to something less aggressive? 
Or you can remove this line at all like it is done in "DansLeRuSH-Dark.xml".

**Review by mere-human — COMMENTED** [review]
Overall, looks good!
Except some literals that are red. See my comment.

image

File: PowerEditor/installer/themes/Obsidian.xml (hunk 1 above)
Comment (victor-istomin) [reply PowerEditor/installer/themes/Obsidian.xml]: Thanks, done! 

image

**Review by mere-human — APPROVED** [review]
Looks good to me.
image

**Comment by victor-istomin** [comment]
> Looks good to me.
Just curious, why `true` and `null` tokens treated as ERROR in your sample. I didn't manage to reproduce it in my example.

**Comment by mere-human** [comment]
> > Looks good to me.
> 
> Just curious, why `true` and `null` tokens treated as ERROR in your sample. I didn't manage to reproduce it in my example.

I don't know either.
Here is the file content, you can try it: https://pastebin.com/yU6jyR3R

What I did was just copy the raw content of the file from this PR and paste into `AppData\Roaming\Notepad++\themes\Obsidian.xml`

Also, for some reason, initially it was detected as Normal Text so I had to set it to JSON from the _Language_ menu.
Maybe, some issues with my local install.

**Comment by victor-istomin** [comment]
> I don't know either.
> Here is the file content, you can try it: https://pastebin.com/yU6jyR3R

Maybe, some updated/outdated JSON lexer issue. In any case, it looks unrelated to themes coloring :)

Thanks a lot for the review! 
image

**Pull request closed without merging**